### ISSUE TITLE
Addition of Shutter Service

### ISSUE THREAD

**Issue by rafterk** [kodmax/homebridge-knx-eib#2]
You have done a wonderful job with this.
Do you have any plans on adding the Shutter service?

**Comment by kodmax** [comment]
yes it should be added eventually. Currently I have no shutter device at home so it makes it harder to test the implementation. Perhaps you can provide some practical information:
- which knx commands should be sent/received
- what would be the mapping from homekit device
